readme.md

```diff
@@ -40,17 +31,17 @@ he or she likes, and on the right, three things he or she dislikes.
 
 Switch!
 
-For the remaining 10 minutes, we’ll go around the room and share!
+For the remaining 10 minutes, we'll go around the room and share!
```

Comment:
I replaced them with standard quotes because it was rendering all weird on my computer.